CHANGELOG.md

```diff
@@ -1,5 +1,18 @@
 # Changelog
 
+## [Version 0.4.0 (Public Preview)](https://github.com/awslabs/aws-mysql-jdbc/releases/tag/0.4.0) - 2021-12-14
+
+### Added
+* Connection plugin support through Connection Plugin Manager. More details can be found [here](https://github.com/awslabs/aws-mysql-jdbc#connection-plugin-manager).
+* Connection plugin, [Enhanced Failure Monitoring](https://github.com/awslabs/aws-mysql-jdbc/#enhanced-failure-monitoring), to reduce the time required to detect connection failure.
```

Comment:
I don't think we have to mention connection plugin again.
```suggestion
* [Enhanced Failure Monitoring](https://github.com/awslabs/aws-mysql-jdbc/#enhanced-failure-monitoring) to reduce the time required to detect connection failure.
```

Reply:
It doesn't necessarily "reduce the time required", it kind of depends on user settings.
I prefer something like "[Enhanced Failure Monitoring](https://github.com/awslabs/aws-mysql-jdbc/#enhanced-failure-monitoring) for more sophisticated failure detection."